File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L6/09/Level6.ts
import { Scanner } from 'readline-sync';

function main(): void {
  const in: Scanner = new Scanner();
  let radius: number, height: number;

  // Get input
  console.log('Enter the radius and length of a cylinder: ');
  radius = in.nextDouble();
  height = in.nextDouble();

  count(radius, height);
}

function count(rad: number, height: number): void {
  console.log(`The area is ${Math.pow(rad, 2) * Math.PI}`);
  console.log(`The volume of the cylinder is ${Math.pow(rad, 2) * Math.PI * height}`);
}

main();

File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L6/09/Level6.py
import math

def main():
    radius, height = map(float, input("Enter the radius and length of a cylinder: ").split())
    count(radius, height)

def count(rad, height):
    print(f"The area is {math.pow(rad, 2) * math.pi}")
    print(f"The volume of the cylinder is {math.pow(rad, 2) * math.pi * height}")

if __name__ == "__main__":
    main()

